src/constants/tokens.ts

```diff
@@ -357,6 +413,13 @@ export const WRAPPED_NATIVE_CURRENCY: { [chainId: number]: Token | undefined } =
     'CELO',
     'Celo native asset'
   ),
+  [SupportedChainId.BOBA]: new Token(
+    SupportedChainId.BOBA,
+    '0x4200000000000000000000000000000000000006',
+    18,
+    'WETH',
```

Comment:
this is the ETH token on L2,
wrapped Ether on L2 is at '0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000'

can we check how optimism handles the native token on interface? (note: the native token acts similar, however the difference being optimism has WETH on 0x42...6 and ETH on 0xDead..000
boba has WETH on 0xDead..000 and ETH on 0x42...6)

Reply:
I already update this for Eth into our network, also i double check the weth url, it´s correct now both both networks (Goerli/Boba)